apps/web/components/Organization/Wrapper/Themes/MannaudstorgTheme/MannaudstorgFooter.tsx

```diff
@@ -0,0 +1,138 @@
+                    paddingBottom={5}
+                    marginBottom={5}
+                    borderColor="blueberry300"
+                    borderBottomWidth="standard"
+                  >
+                    <Box marginRight={4} className={styles.logo}>
+                      {logoSrc ? (
+                        <img src={logoSrc} width={64} height={64} alt="merki" />
+                      ) : (
+                        <Logo iconOnly width={64} height={64} />
+                      )}
+                    </Box>
+                    <Text color="white" variant="h2" marginTop={[4, 4, 0]}>
+                      <Hyphen locale={activeLocale}>{title}</Hyphen>
+                    </Text>
+                  </Box>
+                </GridColumn>
+              </GridRow>
+              {(contactLink || phone) && (
+                <GridRow>
+                  <GridColumn
+                    span={['12/12', '12/12', '7/12']}
+                    paddingBottom={6}
+                  >
+                    <Stack space={2}>
+                      {contactLink && (
+                        <>
+                          <Text variant="h4" color="white">
+                            {headerText}
+                          </Text>
+                          <LinkContext.Provider
+                            value={{
+                              linkRenderer: (href, children) => (
+                                <Link
+                                  href={href}
+                                  color="white"
+                                  underline="normal"
+                                  underlineVisibility="always"
+                                >
+                                  {children}
+                                </Link>
+                              ),
+                            }}
+                          >
+                            <Text color={'blue600'}>
+                              <a href={contactLink}>{linkText}</a>
+                            </Text>
+                          </LinkContext.Provider>
+                        </>
+                      )}
+                      {phone && (
+                        <LinkContext.Provider
+                          value={{
+                            linkRenderer: (href, children) => (
+                              <Link
+                                href={href}
+                                color="white"
+                                underline="normal"
+                                underlineVisibility="always"
+                              >
+                                {children}
+                              </Link>
+                            ),
+                          }}
+                        >
+                          <Inline space={1}>
+                            <Text color="white">
+                              {activeLocale === 'is' ? 'Sími:' : 'Telephone:'}
```

Comment:
Not requesting changes necessarily, just food for thought.
What if f.x. Polish gets added to locales in the future?

Reply:
You are correct, this is something on the backlog that we will address at a later point.